mlflow/ml-package-versions.yml

```diff
@@ -579,6 +582,7 @@ transformers:
       # Installing `transformers==4.34.1` with `datasets` results in installing `huggingface_hub==0.17.3`.
       # `accelerate>=0.32.0` and `peft>=0.13.0` are incompatible with `huggingface_hub==0.17.3`.
       "== 4.34.1": ["accelerate<0.32.0", "peft<0.13.0"]
+      "== dev": ["tf-keras"]
```

Comment:
transformer-dev is not compatible with keras 3 (but it will install keras 3 by default if no keras version restriction),
and transformer-dev with keras 2 has werid bug:
https://github.com/mlflow/mlflow/actions/runs/11227412450/job/31209604569#step:12:451

so I use `tf-keras` instead